clients/src/main/java/org/apache/kafka/clients/consumer/internals/CoordinatorRequestManager.java

```diff
@@ -122,6 +123,22 @@ NetworkClientDelegate.UnsentRequest makeFindCoordinatorRequest(final long curren
         });
     }
 
+    /**
+     * Handles the disconnection of the current coordinator.
+     * This method checks if the given exception is an instance of {@link DisconnectException}.
+     * If so, it marks the coordinator as unknown, indicating that the client should
+     * attempt to discover a new coordinator.
+     *
+     * @param exception     The exception that caused the coordinator to be marked as unknown.
+     *                      This is expected to be an instance of {@link DisconnectException}.
```

Comment:
this param description is a bit confusing. This is the error to handle, received in a request response (not really expecting it to be of any type. If it's DisconnectedException, it will mark the coordinator unknown, or no-op for any other error type)